scripts/dockerdev.sh

```diff
@@ -25,11 +26,7 @@ fi
 
 dockerdev () {
     local container_name=bitbox-wallet-dev
-
-    if ! $RUNTIME images | grep -q bitbox-wallet-app; then
-        echo "No bitbox-wallet docker image found! Maybe you need to run 'make dockerinit'?" >&2
-        exit 1
-    fi
+    local image_name=bitbox-wallet-app
```

Comment:
should be shiftcrypto/bitbox-wallet-app, shiftcrypto/ is part of the image name.